packages/ui/src/council/components/Activities/CandidacyWithdrawContent.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+import { ActivityContentComponent } from '@/common/components/Activities/ActivityContent'
+import { CandidacyWithdrawActivity } from '@/council/types/CouncilActivities'
+
+export const CandidacyWithdrawContent: ActivityContentComponent<CandidacyWithdrawActivity> = ({ activity }) => (
+  <>Council candidate {activity.candidateHandle} has resigned.</>
```

Comment:
I'd put the candidate handle inside of a `MemberModalLink` component like in other activities that display a member. I also considered linking to their candidacy, but I guess that's pretty much pointless given that they've resigned.